config/initializers/slack.rb

```diff
@@ -0,0 +1,4 @@
+Slack = Slack::Notifier.new ENV['slack_webhook_url']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.